crates/lakefs/src/storage.rs

```diff
@@ -121,14 +121,11 @@ mod tests {
     use maplit::hashmap;
     use serial_test::serial;
 
-    struct ScopedEnv {
-        vars: HashMap<std::ffi::OsString, std::ffi::OsString>,
-    }
+    struct ScopedEnv {}
```

Comment:
the compiler claimed this field was never read -- if the CI passes we could remove the entire structure I think (but I wanted to make the diff smaller to see initially)